code/modules/research/designs/misc_designs.dm

```diff
@@ -419,6 +429,26 @@
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/rcd_loaded
+	name = "Rapid Construction Device"
+	desc = "A tool that can construct and deconstruct walls, airlocks and floors on the fly."
+	id = "rcd_loaded"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 60000, MAT_GLASS = 5000)  // costs more than what it did in the autolathe, this one comes loaded.
+	build_path = /obj/item/construction/rcd/loaded
+	category = list("Equipment")
+	departmental_flags =  DEPARTMENTAL_FLAG_ENGINEERING
+
+/datum/design/rpd
+	name = "Rapid Pipe Dispenser"
+	desc = "A tool that can construct and deconstruct pipes on the fly."
+	id = "rpd"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 75000, MAT_GLASS = 37500)
+	build_path = /obj/item/construction/rcd/loaded
```

Comment:
Loaded?

Reply:
Basically it cost more than in the Lathe, but come already filled with Matter-Units.

Reply:
How is the RPD loaded?

Reply:
Sorry, going to fix that now

Reply:
Done